### PR TITLE
Add certification guide

### PR DESCRIPTION
**Pull request by xpepermint**

### CODE REVIEW

File: docs/certification/conventions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Conventions
+
+The 0xcert Framework provides some basic schema conventions which you can use in your projects.
+
+| Issue | Id | Description
+|-|-|-
+| [88](https://0xcert.org/conventions/xcert-schema.html) | xxxx | JSON Schema definition describing asset schema capabilities.
```
Comment (fulldecent) [inline docs/certification/conventions.md]: Link broken

Comment (fulldecent) [reply docs/certification/conventions.md]: https://0xcert.org/conventions/xcert-schema.html

Comment (fulldecent) [reply docs/certification/conventions.md]: What is XXXX

**Comment by fulldecent** [comment]
This PR is mostly reorganizing the documentation and hosting for conventions.

I think of the conventions hosting as an API endpoint. This made sense for it to be on its own subdomain. I'm not sure what the motivation is to move that onto the 0xcert.org main domain name. But either way it's not a big deal.

File: docs/certification/conventions.md (hunk 1 above)
Comment (xpepermint) [reply docs/certification/conventions.md]: This PR is tagged with `work in progress`

**Comment by xpepermint** [comment]
@fulldecent 
- I requested a review for the `.md` files because the content is a complete rewrite.
- Regarding reorganization and conventions, The new 0xcert Framework will have 3 components - 0xcertAPI, Certification, Connectors. I think it would be good to have only one documentation for the whole 0xcert and conventions in one of the chapters. This month we are moving everything into one place - one `0xcert.org`.

**Comment by fulldecent** [comment]
Note to self: build is failing only because URLs are not yet published at the new location.

Example: https://0xcert.org/conventions/ethereum/token-mock.json

File: packages/0xcert-vue-plugin/README.md [hunk 2]
```diff
@@ -43,6 +49,16 @@ const client = this.$0xcert; // 0xcert client
 const provider = this.$0xcert.provider; // current provider
 ```
 
-The [0xcert Framework](https://docs.0xcert.org) is a free and open-source JavaScript library that provides tools for building powerful decentralized applications. Please refer to the [official documentation](https://docs.0xcert.org) for more details.
+VueJS and NuxtJS use Webpack for compiling modules. Webpack can sometimes render the source code in a strange way, ignoring code conditions, thus you may encounter the "missing module" error. The fix for this is to set `node.{module-name} = 'empty'` property in the webpack configuration. The example below shows how you should do this in NuxtJS:
```
Comment (fulldecent) [inline packages/0xcert-vue-plugin/README.md]: -> Vue.js, Nuxt.js

**Comment by fulldecent** [comment]
Looks great. Just minor notes above.

File: docs/certification/guide.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+## Certifying assets
+
+An asset in the 0xcert ecosystem can represent various certificates, ranging from ISO certificates to car insurance contracts, academic credentials, agreements and more. The certification process is responsible for generating digital proofs of asset metadata which can be used by third party users to verify asset information without intermediaries.
+
+To create such proof for an arbitrary asset we need to first prepare
+- asset `metadata` object holding asset information and
+- asset `schema` which describes asset context and metadata structure.
+
+From these two objects, we then generate 
+- asset `imporint` string representing a cryptographic footprint of asset metadata generated through the 0xcert AIH algorithm,
```
Comment (MoMannn) [inline docs/certification/guide.md]: imprint

File: docs/certification/guide.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+```sh
+$ npm i --save @0xcert/cert
+```
+
+On our official open-source [GitHub repository](https://github.com/0xcert/framework), we also host a compiled and minimized JavaScript files that can be directly implemented into your application or website. Please also refer to the [API Reference](/api/core.html) section to learn more about certification.
+
+### Defining asset metadata
+
+Asset metadata represents a digital object which holds asset information. It is described in the form of a common JSON object, which conforms to [RFC-7159](https://en.wikipedia.org/wiki/JSON specification.
```
Comment (MoMannn) [inline docs/certification/guide.md]: Missing `)`.

File: docs/certification/guide.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+  "$evidence": "https://troopersgame.com/dog-evidence.json",
+  ...
+}
+```
+
+### Defining asset schema
+
+A digital asset in the 0xcert Protocol employs a JSON Schema to describe asset context and data object structure. The asset schema allows for machine-level processing of asset information and thus makes each asset interoperable at the application layer.
+
+Asset schema is a specifically designed JSON object, which conforms to [RFC-7159](https://en.wikipedia.org/wiki/JSON) and follows the mapping format defined by the [JSON Schema](http://json-schema.org) specification. The 0xcert Protocol supports only a subset of JSON Schema specification which is described by the [Xcert JSON Schema](). You should always refer to this JSON Schema subset when writing asset schemas for your projects.
```
Comment (MoMannn) [inline docs/certification/guide.md]: Missing Xcert JSON Schema link

File: docs/certification/guide.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+We incentivize the community to agree on standard schema conventions. Schema conventions should be proposed and included in the official 0xcert repository on Github. The 0xcert Framework already provides some basic [schema conventions]() which you can use. For the purpose of this guide, we will define a minimal custom asset schema which describes a context and properties of the metadata which we created earlier.
```
Comment (MoMannn) [inline docs/certification/guide.md]: missing schema convention link

File: docs/certification/guide.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+
+### Creating asset evidence, imprint and schema ID
+
+The previous section explained how a context and metadata structure information are attached to an asset. In this chapter, we dive into the core mistery of certification and calculate the `imprint`, `evidence` and `schema ID` artifacts. These are the main products of the certification process defined by the 0xcert Protocol which allows for third party users to verify asset metadata information without an intermediary.
+
+An imprint represents a cryptographic footprint of a complete asset metadata object. It is calculated through the SXH2 algorithm and represents a mathematical proof of asset metadata existence, where at the same time ensures metadata confidentiality. The imprint is one of the decentralized artifacts of the certification process and should thus be stored on immutable storage like permissionless public blockchain (e.g. Ethereum) where trust is not an issue and third-party users can autonomously read data.
```
Comment (MoMannn) [inline docs/certification/guide.md]: Maybe cryptographic fingerprint?

File: docs/certification/api.md [hunk 8]
```diff
@@ -0,0 +1,431 @@
...
+  name: 'John',
+};
+
+// calculate imprint
+const imprint = await cert.calculate(metadata, evidence);
+```
+
+### disclose(metadata, paths)
+
+An `asynchronous` class instance `function` which generates an `evidence` for the subset of the metadata. The selected properties are provided as object `paths`.
```
[the remark this answers is not in the document]
Comment (xpepermint) [reply docs/certification/api.md]: array of strings

**Pull request merged**